pkg/userdata/ubuntu/provider.go

```diff
@@ -246,11 +246,11 @@ write_files:
     mv /etc/fstab.noswap /etc/fstab
     swapoff -a
 
-    export CR_PKG='docker-ce=5:18.09.9~3-0~ubuntu-bionic'
-    export CR_CLI_PKG='docker-ce-cli=5:18.09.9~3-0~ubuntu-bionic'
-    export DEBIAN_FRONTEND=noninteractive
+{{- /* We need to explicitly specify docker-ce and docker-ce-cli to the same version.
+	See: https://github.com/docker/cli/issues/2533 */}}
 
-    apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install -y \
+    DOCKER_VERSION='5:18.09.9~3-0~ubuntu-bionic'
```

Comment:
What's the reason for having the version string starting with `5:` and ending with `~3-0~ubuntu-bionic`? It can be confusing from where this string comes from. It should also work well if we only provide the version.

Reply:
this is the version of the package, I don't know where this convention comes from by I don't think we should only take part of it.